src/test/scala/in/ashwanthkumar/matsya/ClusterConfigTest.scala

```diff
@@ -0,0 +1,39 @@
+package in.ashwanthkumar.matsya
+
+import java.io.File
+
+import com.typesafe.config.ConfigFactory
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers.{be, convertToAnyShouldWrapper}
+
+
+class ClusterConfigTest extends FlatSpec {
+
+  "ClusterConfig Object" should "create ClusterConfig from config " in {
+
+    val config = ConfigFactory.parseFile(new File("src/test/resources/test-cluster-config.conf"))
```

Comment:
Instead of specifying the location this way - you can also use [`Class.getResource`](https://docs.oracle.com/javase/7/docs/api/java/lang/Class.html#getResource(java.lang.String)) instead
```
new File(this.getResource("/test-cluster-config.conf"))
```